stubs/pika/pika/spec.pyi

```diff
@@ -338,9 +338,9 @@ class Exchange(Class):
         INDEX: ClassVar[int]
         def __init__(self) -> None: ...
         @property
-        def synchronous(self): ...
-        def decode(self, encoded, offset: int = ...): ...
-        def encode(self): ...
+        def synchronous(self) -> Literal[False]: ...
+        def decode(self: Self, encoded: bytes, offset: int = ...) -> Self: ...
```

Comment:
`encoded` and `offset` are unused, so technically they take `object`, but there's no reason to provide wrong argument, so it's safer to demand to use the "correct" types here.